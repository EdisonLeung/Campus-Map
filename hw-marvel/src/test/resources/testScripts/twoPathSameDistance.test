# Tests pathfinding on a node with 2 path options which have the same length.
# Should print the path with the lexicographically least node (if nodes the same, then
# compares the edges

# Create graph
CreateGraph sameDistance

# Add nodes
AddNode sameDistance a
AddNode sameDistance b

# Add edges
AddEdge sameDistance a b ABa
AddEdge sameDistance a b ABb

# test pathfinding from a to b
FindPath sameDistance a b