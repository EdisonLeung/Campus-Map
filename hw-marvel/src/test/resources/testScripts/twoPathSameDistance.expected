# Tests pathfinding on a node with 2 path options which have the same length.
# Should print the path with the lexicographically least node (if nodes the same, then
# compares the edges

# Create graph
created graph sameDistance

# Add nodes
added node a to sameDistance
added node b to sameDistance

# Add edges
added edge ABa from a to b in sameDistance
added edge ABb from a to b in sameDistance

# test pathfinding from a to b
path from a to b:
a to b via ABa